packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -230,7 +235,7 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   }
 
   /* @conditional-compile-remove(video-background-effects) */
-  if (useIsParticularSidePaneOpen('videoeffects') && props.errorBarProps) {
+  if ((useIsParticularSidePaneOpen('videoeffects') || !isCameraOn) && props.errorBarProps) {
```

Comment:
is this supposed to be !isCameraOn?
The title indicates that it should dismiss when camera is off, but this is setting an activemessage when camera is off.

I see that in videoEffectsPane.tsx line 149 new change it does what I expected.
Correct me if I'm wrong here.

Reply:
Yeah, so in this scenario, if `videoeffectsSidepane` is `open` and `camera` is `off` we are filtering out the video effects error from latest errors.
For videoeffectsPane, if the `videoeffectsError` exists and `camera is on` we are showing the error.